src/Serialization/Hash.purs

```diff
@@ -48,21 +49,21 @@ instance Ord Ed25519KeyHash where
   compare = compare `on` ed25519KeyHashToBytes
 
 instance Show Ed25519KeyHash where
-  show edkh = "(Ed25519KeyHash " <> byteArrayToHex (ed25519KeyHashToBytes edkh)
+  show edkh = "(Ed25519KeyHash " <> rawBytesToHex (ed25519KeyHashToBytes edkh)
     <> ")"
 
 instance ToData Ed25519KeyHash where
-  toData = toData <<< ed25519KeyHashToBytes
+  toData = toData <<< unwrap <<< ed25519KeyHashToBytes
 
 instance FromData Ed25519KeyHash where
-  fromData (Bytes kh) = ed25519KeyHashFromBytes kh
+  fromData (Bytes kh) = ed25519KeyHashFromBytes <<< wrap $ kh
```

Comment:
If you like, you could write this as `ed25519KeyHashFromBytes $ wrap kh`

Reply:
If you say it's nicer, then I will. ;-)